### PR TITLE
HW01_Xingruo Zhang

### PR DESCRIPTION
**Pull request by Xingruo-Z**
The purpose of this repository is to give a brief introduction of myself. I didn't have a personal website, so I chose to share my favorite illustrator with you. I used headers, emphasis, lists, links, and images in this homework. I found [GitHub Guides](https://guides.github.com) and [Pandoc](https://pandoc.org/MANUAL.html#extension-link_attributes) helpful for reference.

Headers, emphasis, lists, and links were easy to add. However, it was quite difficult for me to embed an image. At first, it only showed links every time. After referring to the aforementioned websites and some cheatsheet, I realized that I should use a link ended with an image file format. Then it worked. However, I still had some difficulties changing the size of an image. Using {width=50%}, it changed accordingly in R Markdown Preview, but there was no change on Github. Instead, the code appeared below the image. Also, I forgot to make commits for each line, so I redid it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
+
+I am a MAPSS student in Psychology track. My interests lie in cultural psychology and behavioral science. I graduated with a Bachelor degree of Business Management from the Chinese University of Hong Kong, Shenzhen this year. Currently, I am studying remotely from China. 
+
+* Hobbies
+  1. Travelling
+  2. Watching crime dramas
+  
+My favoriate illustrator: **Lost 7**
+
+Please see his page(https://lost7.zcool.com.cn)
```
Comment (Donghyun-Kang-Soc) [inline README.md:12]: I would expect to see something like `[**Lost 7**](https://lost7.zcool.com.cn)` here, which would like [**Lost 7**](https://lost7.zcool.com.cn) given that the purpose of doing this assignment is to experiment with the markdown syntax.

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+
+![A Picture by *Lost 7*](https://c-ssl.duitang.com/uploads/item/201611/04/20161104002329_sBY8V.jpeg)
```
Comment (Donghyun-Kang-Soc) [inline README.md:14]: I did not want to repeat what Dr. Soltoff said, but it could have been resized.

File: README.md [hunk 3]
```diff
@@ -1 +1,14 @@
-# hw01
+# HW01 Self-Introduction
+## Name: Xingruo Zhang
+
+I am a MAPSS student in Psychology track. My interests lie in cultural psychology and behavioral science. I graduated with a Bachelor degree of Business Management from the Chinese University of Hong Kong, Shenzhen this year. Currently, I am studying remotely from China. 
+
+* Hobbies
+  1. Travelling
```
Comment (Donghyun-Kang-Soc) [inline README.md:7]: A good combination of the unordered list and the ordered list.

**Review by Donghyun-Kang-Soc — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |     X       |            |
| **Coding strategy**      |           |      X       |          |
| **Presentation**            |           |      X      |          |
| **Achievement, creativity** |           |      X        |            |
| **Reproducibility**         |    X        |             |            |

Remarks:
* You used different headers levels, a combination of an ordered list and an unordered list, emphasis, and embedded an image.
* Please check out my comment about embedding links with texts. 
* I wish you had experimented with more syntax elements (and introduced yourself a little more). But I also hope you will use other R markdown elements next time you submit assignments throughout the quarter.

File: README.md [hunk 4]
```diff
@@ -1 +1,14 @@
-# hw01
+# HW01 Self-Introduction
+## Name: Xingruo Zhang
```
Comment (ogill-stuart) [inline README.md:2]: I like the way you made your headers really clear.

File: README.md [hunk 5]
```diff
@@ -1 +1,14 @@
...
+
+I am a MAPSS student in Psychology track. My interests lie in cultural psychology and behavioral science. I graduated with a Bachelor degree of Business Management from the Chinese University of Hong Kong, Shenzhen this year. Currently, I am studying remotely from China. 
+
+* Hobbies
+  1. Travelling
+  2. Watching crime dramas
```
Comment (ogill-stuart) [inline README.md:8]: I love crime tv shows as well!

**Pull request closed without merging**